articles/how-to-python-qdk-local.md

```diff
@@ -77,7 +77,11 @@ Once imported, the `qsharp` package allows Q# namespaces to appear as Python mod
     ```output
     Hello, quantum world!
     ```
-    
+
+> [!NOTE]
+> If you hit an error `Unable to load shared library 'Microsoft.Quantum.Simulator.Runtime' or one of its dependencies`, then you need to install the `libomp` library, as the OpenMP support library is required by the quantum simulator of the QDK. On Ubuntu, you can do so by running `sudo apt install libomp5-11`, or `yum install libomp` on CentOS. For other distributions, please refer to your particular package manager.  
+On macOS you can install `libomp` by running `brew install libomp`.
```

Comment:
This seems to be one of the possibilities along with Ubuntu and CentOS, yet it's a separate sentence. I suggest saying something like:
You can install 'libomp' by running '...' on Ubuntu, '...' on Cent OS, '..' on macOS. For other distributions...

Reply:
Changed.